src/components/css/react-dates@20.1.0-fix.css

```diff
@@ -1,3 +1,13 @@
 .DateInput_input {
     box-sizing: border-box;
-}
\ No newline at end of file
+}
+
+.DayPickerNavigation__verticalDefault {
+    text-align: center;
+    height: initial;
+    padding: 10px 0px;
+}
```

Comment:
Adding `box-sizing: border-box;` will take the border width into account when calculating the `50%` size for the buttons, resulting in arrows being side by side.

![image](https://user-images.githubusercontent.com/8092993/69176789-63af5180-0ad4-11ea-956e-8469b4e9b2e4.png)